release-notes/Cabal-3.14.1.1.md

```diff
@@ -0,0 +1,6 @@
+Cabal and Cabal-syntax 3.14.1.1 changelog and release notes
+---
+
+### Significant changes
+
+- Support for GHC 9.12 [10671](https://github.com/haskell/cabal/pull/10671)
```

Comment:
Maybe add a link to https://github.com/haskell/cabal/issues/10670?..

Reply:
why?

Reply:
We usually reference both the issue and the PR in changelos (when we have both, of course). I think it's tidier that way. But if you don't think it's worth it, I'm fine with that.

Reply:
Resolving then...